etc/kayobe/kolla/inventory/overcloud-components.j2

```diff
@@ -0,0 +1,225 @@
+# This inventory section provides a mapping of the components to top level
```

Comment:
Do we need this, given that it's only deployed to the compute nodes when kuryr is enabled?

Reply:
kolla-ansible will generate an incorrect CD_INITIAL_CLUSTER environmental variable (it incorrectly includes the compute nodes when kuryr is disabled)

Reply:
Is there an upstream bug for that?